### ISSUE TITLE
PropertyFieldToggleWithCallout disabled style not applied

### ISSUE THREAD

**Issue by deanbot** [pnp/sp-dev-fx-property-controls#240]
#### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question

#### Version

Please specify what version of the library you are using: 1.17.0

#### Expected / Desired Behavior / Question

A disabled ToggleWithCallout appears, visually, disabled.

#### Observed Behavior
Disabled ToggleWithCallout appears selectable. 

See disabled OOB toggle above disabled ToggleWithCallout:

![image](https://user-images.githubusercontent.com/2632373/80616990-28139e80-8a07-11ea-9d1f-d5f8c76bea55.png)

![image](https://user-images.githubusercontent.com/2632373/80617031-35308d80-8a07-11ea-835b-846fa084551a.png)

Perhaps this is an issue with ouifr v 5.131.0? maybe updating to the latest used in the default generator (v6.189.2) will resolve.

#### Steps to Reproduce

```ts
PropertyFieldToggleWithCallout("foo", {
                  label: "Test Disabled Toggle",
                  calloutContent: React.createElement(
                    "p",
                    { style: { width: "300px" } },
                    "lorem ipsum..."
                  ),
                  calloutTrigger: CalloutTriggers.Click,
                  key: "fooKey",
                  disabled: true
                }),
```

**Comment by AJIXuMuK** [comment]
Hi @deanbot!
Thank you for reporting the issue!
In my tests `toggle` is disabled. The only issue I found is the label for the toggle (with information icon) is not disabled if `disabled=true`.
This issue has been addressed in #256 and will be available in the next release.
You can test the functionality now using [beta version](https://pnp.github.io/sp-dev-fx-property-controls/beta/)

**Comment by AJIXuMuK** [comment]
The version has been released.
